src/app/client/src/app/modules/workspace/services/batch/batch.service.ts

```diff
@@ -0,0 +1,228 @@
+import { Injectable, Input } from '@angular/core';
+import { ConfigService, ServerResponse } from '@sunbird/shared';
+import { SearchService, SearchParam, LearnerService, ContentService, UserService } from '@sunbird/core';
+import { Ibatch } from './../../interfaces/batch';
+import { Observable } from 'rxjs/Observable';
+import 'rxjs/add/operator/map';
+import 'rxjs/add/observable/throw';
+import * as _ from 'lodash';
+/**
+ * Service for batch
+ */
+@Injectable()
+export class BatchService {
+  /**
+  *  orgIds
+  */
+  orgIds: string;
+  /**
+  *  rootOrgId
+  */
+  rootOrgId: string;
+  /**
+   * Reference of user service.
+   */
+  public user: UserService;
+
+  /**
+   * Reference of content service.
+   */
+  public content: ContentService;
```

Comment:
In this file also change propertyName of the service like

public contentService: ContentService